### PR TITLE
Testnet node release 12 Jan 2023

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about the features of the testnet node release on 12 January 2023.

## Changes

Release notes, downloads

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/resources/release-notes.rst [hunk 1]
```diff
@@ -11,6 +11,29 @@ Release Notes - Testnet
...
+January 12, 2023
+
+Concordium Node 5.1.3
+---------------------
+
+Concordium node version 5.1.3 introduces the following new features and improvements:
+
+- Improvements were made to allow greater concurrency with transaction processing. (Checking transaction signatures is done without acquiring the global state lock.)
+
+- Blocks are relayed earlier. In particular this means that blocks are now processed in two steps: block receive and block execute. Blcok receive performs verification of block metadata while the block execute adds the block to the tree. Blocks are now enqueued in the outgoing message queue in between the two steps.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:23]:
```suggestion
- Blocks are relayed earlier. This decreases the time it takes for the network to become aware of a block.
```

File: source/mainnet/net/resources/release-notes.rst [hunk 2]
```diff
@@ -11,6 +11,29 @@ Release Notes - Testnet
...
+- Removed the configuration option ``no_rebroadcast_consensus_validation``.
+
+- Changes were made to avoid deadlocks during node shutdown in specific scenarios.
+
+- The node will now shut down to start if an error occurs in a required service (e.g., grpc server). In particular, the node will shut down if a required service could not be started.
+
+- Added timeout to downloading out of band catchup files when block indices and catch-up chunk files are specified by an URL. The timeout is controlled by the option ``--download-blocks-timeout`` (environment variable ``CONCORDIUM_NODE_CONSENSUS_DOWNLOAD_BLOCKS_TIMEOUT``) and defaults to 5 minutes. Timeout is now five minutes per chunk instead of waiting indefinitely.
+
+- Removed the "instrumentation" feature of the node and built the node with Prometheus support enabled by default.
+
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:34]:
```suggestion
```
This was an internal thing so I would not include it in this changelog.

File: source/mainnet/net/resources/release-notes.rst [hunk 3]
```diff
@@ -11,6 +11,29 @@ Release Notes - Testnet
...
+Concordium Node 5.1.3
+---------------------
+
+Concordium node version 5.1.3 introduces the following new features and improvements:
+
+- Improvements were made to allow greater concurrency with transaction processing. (Checking transaction signatures is done without acquiring the global state lock.)
+
+- Blocks are relayed earlier. In particular this means that blocks are now processed in two steps: block receive and block execute. Blcok receive performs verification of block metadata while the block execute adds the block to the tree. Blocks are now enqueued in the outgoing message queue in between the two steps.
+
+- Removed the configuration option ``no_rebroadcast_consensus_validation``.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:25]:
```suggestion
- Removed the configuration option ``no_rebroadcast_consensus_validation``. This option (which was used for testing only and was disabled by default) made the node rebroadcast blocks before doing any validation.
```

File: source/mainnet/net/resources/release-notes.rst [hunk 4]
```diff
@@ -11,6 +11,29 @@ Release Notes - Testnet
...
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 12, 2023
+
+Concordium Node 5.1.3
+---------------------
+
+Concordium node version 5.1.3 introduces the following new features and improvements:
+
+- Improvements were made to allow greater concurrency with transaction processing. (Checking transaction signatures is done without acquiring the global state lock.)
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:21]:
```suggestion
- Improvements were made to allow greater concurrency with transaction processing.
```
Some other checks are also done without the lock now. I would not go into those details.

File: source/mainnet/net/resources/release-notes.rst (hunk 2 above)
Comment (abizjak) [reply source/mainnet/net/resources/release-notes.rst:34]: It is a build-time configuration, it is not a feature that was ever enabled for any of the distributions builds.

**Review by abizjak — APPROVED** [review]
Looks good. I have some suggestions for improving the changelog.

I checked the hash of the .deb package as well.

File: source/mainnet/net/installation/previous-node-downloads.rst [hunk 5]
```diff
@@ -33,6 +35,8 @@ Windows - Mainnet and Testnet
 macOS - Mainnet and Testnet
 ===========================
 
+`macOS 5.0.6 <https://distribution.concordium.software/macos/signed/concordium-node-5.0.6-0.pkg>`_
```
Comment (mh-concordium) [inline source/mainnet/net/installation/previous-node-downloads.rst:38]: I assume this should be replaced with 5.0.7

**Review by mh-concordium — CHANGES_REQUESTED** [review]
I have one minor comment.

File: source/mainnet/net/installation/previous-node-downloads.rst (hunk 5 above)
Comment (dg-concordium) [reply source/mainnet/net/installation/previous-node-downloads.rst]: Fixed.

**Review by mh-concordium — APPROVED** [review]

**Pull request merged**